libs/Facilitator/Facilitator.js

```diff
@@ -0,0 +1,387 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const BN = require('bn.js');
+const web3 = require('web3');
+const Contracts = require('../../libs/Contracts');
+const Utils = require('../utils/Utils.js');
+
+/**
+ * Class to facilitate stake and mint.
+ *
+ * @class
```

Comment:
Not sure this is required for classes. I think this is supposed to be used with constructor functions to mark them as classes.